### ISSUE TITLE
Wrong postscript file

### ISSUE THREAD

**Issue by fabitom** [marklundeberg/dokuwiki-plugin-latex#10]
In some cases latex genarage empty png.
I found that dvips generate wrong ps file in some cases (tested on PSTricks package).
Convert can't read that file

```
convert-im6.q16: geometry does not contain image `test.ps' @ warning/attribute.c/GetImageBoundingBox/247.
# identify test.png
test.png PNG 1x1 1x1-1-1 8-bit sRGB 123B 0.000u 0:00.000
```

It seems dvips bug [https://groups.google.com/forum/#!msg/comp.text.tex/VvDvcQlCaag/X4pyuX999nIJ](url)

I needed to remove -E (Try to create EPSF) option hard coded in class.latexrender.php and all works ok.

**Comment by u07** [comment]
I agree. Had a problem with that "-E" key too. Spent a day hunting it.
https://tex.stackexchange.com/questions/518409/
Still not sure, what was the original idea of including it in the command line.
